app/src/main-process/notifications.ts

```diff
@@ -35,6 +36,10 @@ export function initializeDesktopNotifications() {
   initializeNotifications({ toastActivatorClsid: windowsToastActivatorClsid })
 }
 
+export function terminateNotificationSystem() {
+  terminateNotifications()
+}
```

Comment:
Let's use the same `DesktopNotifications` suffix:
```suggestion
export function terminateDesktopNotifications() {
  terminateNotifications()
}
```